src/test/java/seedu/address/logic/commands/PropertyCommandTestUtil.java

```diff
@@ -0,0 +1,16 @@
+package seedu.address.logic.commands;
+
+/**
+* Contains helper methods for testing property commands.
+*/
+public class PropertyCommandTestUtil {
+    public static final String VALID_NAME_PROPERTY1 = "Heng Mui Keng Terrace HDB";
+    public static final String VALID_NAME_PROPERTY2 = "ORCHARD ROAD CONDO";
+    public static final String VALID_NAME_PROPERTY3 = "West Coast Street 28 HDB";
+
+    public static final String VALID_PRICE_PROPERTY1 = "1000000";
+    public static final String VALID_PRICE_PROPERTY2 = "780,000";
+    public static final String VALID_PRICE_PROPERTY3 = "2 Million";
```

Comment:
I think you're not the one who implemented this but we should probably standardise the price inputs and format for both buyer and property, and maybe we should be parsing these into a value instead of having the price as a String for future use?

Reply:
no it was me haha. I was thinking we could make the parsing smarter because this will have a better UX, but i should remove it for now